src/main.py

```diff
@@ -75,6 +82,9 @@ def execute_block(block_name):
 
         cleaned_params[name] = body['params'][name]
 
+    if is_resuming and 'state' not in body:
+        return 'State must be sent when resuming', HTTPStatus.BAD_REQUEST
+
```

Comment:
Duplicate test: Already present at lines 60-61

Reply:
Better to be safe than sorry, right? 😅 Will remove!